src/plugins/userVoiceShow/index.tsx

```diff
@@ -93,22 +108,48 @@ export default definePlugin({
         );
     },
 
+    patchUserList: ({ user }: UserProps, dmList: boolean) => {
+        if (!settings.store.showVoiceActivityIcons) return null;
+
+        return (
+            <ErrorBoundary noop>
+                <VoiceActivityIcon user={user} dmChannel={dmList} />
+            </ErrorBoundary>
+
+        );
+    },
+
     patches: [
-        // above message box
         {
             find: ".popularApplicationCommandIds,",
             replacement: {
                 match: /applicationId:\i\.id}\),(?=.{0,50}setNote:\i)/,
                 replace: "$&$self.patchPopout(arguments[0]),",
             }
         },
-        // below username
         {
             find: ".Messages.MUTUAL_GUILDS_WITH_END_COUNT", // Lazy-loaded
             replacement: {
                 match: /\.body.+?displayProfile:\i}\),/,
+                // paste my fancy custom button below the username
                 replace: "$&$self.patchModal(arguments[0]),",
             }
+        },
+        {
+            // Patch Member List
+            find: ".MEMBER_LIST_ITEM_AVATAR_DECORATION_PADDING)",
+            replacement: {
+                match: /avatar:\(\(/,
```

Comment:
```suggestion
                match: /avatar:\i\(\i,\i\)/,
```

This broke I fixed it this way

Reply:
I forgot to push a fix i did two week ago.
i changed it to 
```js
match: /avatar:(\i){1,2}/
```